spec/models/piece_spec.rb

```diff
@@ -50,4 +50,51 @@
       king.errors.clear
     end
   end
+  describe "#move_to!" do
+    context "when the tile is empty" do
+      it "moves to the coordinates" do
+        king = game.kings.first
+        king.update_attributes(x_position: 1, y_position: 1)
```

Comment:
you also have a pawn in this position at the same time. i would suggest either wiping the board clean using `empty_board = create(:game)`  then use `empty_board.pieces.delete` to remove all the pieces and create new ones individually. I checked this by putting binding.pry on line 57, and saw: 
```
 pry(#<RSpec::ExampleGroups::Piece::MoveTo::WhenTheTileIsEmpty>)> game.pieces.where(x_position: 1, y_position: 1)
=> [#<King id: 1040, x_position: 1, y_position: 1, piece_type: "King", color: true, game_id: 33, user_id: 67, captured: false, created_at: "2015-12-01 06:31:11", updated_at: "2015-12-01 06:32:27", image_name: "white-king.png">,
 #<Pawn id: 1026, x_position: 1, y_position: 1, piece_type: "Pawn", color: true, game_id: 33, user_id: 67, captured: false, created_at: "2015-12-01 06:31:11", updated_at: "2015-12-01 06:31:11", image_name: "white-pawn.png">]
```

you could also just move pieces around on the board to clear them out of your way